src/modules/App/stories/index.stories.js

```diff
@@ -533,3 +534,17 @@ export const userListQuery = () => fitPageSize(
     userListQuery={getCustomPaginatedQuery}
   />
 );
+
+export const multipleFilesMessage = () => fitPageSize(
+  <App
+    appId={appId}
+    userId={multipleFilesMessageUserId}
+    nickname={multipleFilesMessageUserId}
+    showSearchIcon
+    isMentionEnabled
+    isReactionEnabled
+    isTypingIndicatorEnabledOnChannelList
+    isMessageReceiptStatusEnabledOnChannelList
+    config={{ logLevel: 'all', isREMUnitEnabled: true }}
+  />
+);
```

Comment:
What is this for? It seems there are no configs about multiple files message
Could you leave a comment to explain this?

Reply:
Ah this is not relevant to this PR. Ill remove this.

Reply:
removed